specification/web/resource-manager/Microsoft.Web/stable/2016-08-01/WebApps.json

```diff
@@ -12320,10 +12320,23 @@
         ],
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Successfully retrieved source control for web app.",
+            "schema": {
+              "$ref": "#/definitions/SiteSourceControl"
+            }
+          },
+          "201": {
```

Comment:
Does "x-ms-long-running-operation: true" need to be added to this (and the other) operation?